index.html

```diff
@@ -1,110 +1,94 @@
-                            </div>
-                            <div class="feature is-revealing">
-                                <div class="feature-inner">
-                                    <a href="//lsp.sublimetext.io" class="feature-icon">
-                                        <img class="asset-light" src="dist/images/sublime-lsp.png" alt="Sublime LSP">
-                                        <img class="asset-dark" src="dist/images/sublime-lsp.png" alt="Sublime LSP">
-                                    </a>
-                                    <div class="feature-content">
-                                        <a href="//lsp.sublimetext.io">
-                                            <h3 class="feature-title mt-0">Sublime LSP Documentation</h3>
-                                        </a>
-                                        <p class="text-sm mb-0">Sublime Text's community driven implementation for LSP
-                                            support.
-                                        </p>
-                                    </div>
-                                </div>
-                            </div>
-                            <div class="feature is-revealing">
-                                <div class="feature-inner">
-                                    <!-- Hard-code HTTP because the redirect set up by namecheap cannot handle HTTPS. -->
-                                    <a href="http://discord.sublimetext.io" class="feature-icon">
-                                        <img class="asset-light" src="dist/images/discord_1024x1024x32.png"
-                                            alt="Sublime Discord">
-                                        <img class="asset-dark" src="dist/images/discord_1024x1024x32.png"
-                                            alt="Sublime Discord">
-                                    </a>
-                                    <div class="feature-content">
-                                        <a href="http://discord.sublimetext.io">
-                                            <h3 class="feature-title mt-0">Sublime Text Discord (Unofficial)</h3>
-                                        </a>
-                                        <p class="text-sm mb-0">This discord server is a place for discussion about <a
-                                                href="https://sublimetext.com/">Sublime Text</a> and <a
-                                                href="https://sublimemerge.com/">Sublime Merge</a>.
-                                        </p>
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </section>
-        </main>
-    </div>
-    <script src="dist/js/main.min.js"></script>
+         </div>
+      </div>
+      <div class="love-sublime">Made with ❤️ on Sublime Text</div>
```

Comment:
I find these "made with :heart:" kinda cringe and superfluous tbh, but I'm not strictly against it if @TerminalFi likes it. It just doesn't convey any useful information (like, surely you didn't make it with hate or something, but even if you did you probabaly wouldn't tell).

Reply:
ok ok i understand! but unfortunately i am only the developer and you are going to have to speak with my product manager :(

<img width="684" alt="image" src="https://user-images.githubusercontent.com/58625258/215297196-7e146e1e-0789-4ad5-81a8-29daad2d9fa9.png">


can we come compromise maybe? i remove html. i put in a console.log
<img width="307" alt="image" src="https://user-images.githubusercontent.com/58625258/215297161-44d32f74-6abe-45ab-af94-4655131aebe0.png">